From 4effa711a867eaf0f10b38c2bcf4c8c39a00c1ab Mon Sep 17 00:00:00 2001
From: Laurent Vivier <lvivier@redhat.com>
Date: Fri, 14 Aug 2020 16:02:48 -0400
Subject: [PATCH 1/4] redhat: Update hw_compat_8_2

RH-Author: Laurent Vivier <lvivier@redhat.com>
Message-id: <20200814160249.217753-2-lvivier@redhat.com>
Patchwork-id: 98157
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH 1/2] redhat: Update hw_compat_8_2
Bugzilla: 1843348
RH-Acked-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

v5.1.0-rc2 has introduced a new entry in hw_compat_5_0:
  2ebc21216f58 ("hw/pci-host: save/restore pci host config register")

Add it in hw_compat_rhel_8_2

Update hw_compat_8_2 comment as it also includes hw_compat_5_0
Move a blank line

Signed-off-by: Laurent Vivier <lvivier@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/core/machine.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index 6d17d9938d..10fa9b8c75 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -29,7 +29,7 @@
 #include "migration/vmstate.h"
 
 /*
- * The same as hw_compat_4_2
+ * The same as hw_compat_4_2 + hw_compat_5_0
  */
 GlobalProperty hw_compat_rhel_8_2[] = {
     /* hw_compat_rhel_8_2 from hw_compat_4_2 */
@@ -55,6 +55,8 @@ GlobalProperty hw_compat_rhel_8_2[] = {
     /* hw_compat_rhel_8_2 from hw_compat_4_2 */
     { "fw_cfg", "acpi-mr-restore", "false" },
     /* hw_compat_rhel_8_2 from hw_compat_5_0 */
+    { "pci-host-bridge", "x-config-reg-migration-enabled", "off" },
+    /* hw_compat_rhel_8_2 from hw_compat_5_0 */
     { "virtio-balloon-device", "page-poison", "false" },
     /* hw_compat_rhel_8_2 from hw_compat_5_0 */
     { "vmport", "x-read-set-eax", "off" },
@@ -64,9 +66,9 @@ GlobalProperty hw_compat_rhel_8_2[] = {
     { "vmport", "x-report-vmx-type", "off" },
     /* hw_compat_rhel_8_2 from hw_compat_5_0 */
     { "vmport", "x-cmds-v2", "off" },
-
 };
 const size_t hw_compat_rhel_8_2_len = G_N_ELEMENTS(hw_compat_rhel_8_2);
+
 /*
  * The same as hw_compat_4_1
  */
-- 
2.27.0

